runtime/shiden/src/precompiles.rs

```diff
@@ -63,6 +65,31 @@ impl Contains<RuntimeCall> for WhitelistedCalls {
         }
     }
 }
+/// Filter that only allows whitelisted runtime call to pass through dispatch-lockdrop precompile
+pub struct WhitelistedLockdropCalls;
```

Comment:
Might be neat to put this under primitives since it will be the same for both Shiden/Astar.

Reply:
There is dependancy over many runtime pallets (ds, assets, pallet_utility, pallet_balances). I don't think it will be more clean to make it generic over all runtimes

Reply:
I know, but you'd put it as trait bound, same as we do for many types already. E.g. for the precompiles.

Anyways, just a suggestion.